### PR TITLE
fix env from generated doc

### PR DESCRIPTION
**Pull request by NicolasMahe**
fix env from generated doc

### CODE REVIEW

File: packages/cli/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #### Improvements
 
 - [#190](https://github.com/liteflow-labs/liteflow-js/pull/190) Set the name of the process to the log of the deployed endpoint.
+- [#200](https://github.com/liteflow-labs/liteflow-js/pull/200) Fix env from generated doc.
```
Comment (antho1404) [inline packages/cli/CHANGELOG.md:10]: This is a fix, should be in the bug fix section

Comment (antho1404) [inline packages/cli/CHANGELOG.md:10]:
```suggestion
```

File: packages/cli/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
...
 
 #### Bug fixes
```
Comment (antho1404) [inline packages/cli/CHANGELOG.md:12]:
```suggestion
#### Bug fixes
- [#200](https://github.com/liteflow-labs/liteflow-js/pull/200) Fix env from generated doc.
```

**Comment by NicolasMahe** [comment]
@antho1404 changelog changed

**Pull request merged**